client/src/components/layout/AccountPage.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import _ from "lodash";
+import Dropzone from "react-dropzone";
+
+const AccountPage = (props) => {
+ 
+
+  const [user, setUser] = useState(props.user); 
+  const [profilePicture, setProfilePicture] = useState({}); 
+  const handleImageUpload = (acceptedImage) => {
+    setProfilePicture({
+      ...profilePicture,
+      image: acceptedImage[0],
+    });
+  };
+
+  const addProfilePicture = async () => {
+    const newProfilePictureBody = new FormData();
+    newProfilePictureBody.append("image", profilePicture.image);
+    try {
+      const response = await fetch(`/api/v1/users/${profilePicture.id}`, {
+        method: "PATCH",
+        headers: {
+          Accept: "image/jpeg",
+        },
+        body: newProfilePictureBody,
+      });
+      if (!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`);
+      }
+      const body = await response.json();
+      setUser({ ...user, image: body.user.image });
+    } catch (error) {
+      console.error(`Error in addProfilePicture Fetch: ${error.message}`);
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    addProfilePicture();
+  };
+
+  let uploadImagePromptText = "Click Or Drag Here To Edit Profile Image";
+
+  const handleClick = (event) => {
+    uploadImagePromptText = "Click Or Drag Here To Edit Profile Image";
+  }
```

Comment:
unfortunately this function does not add any functionality. remove it. 

your form submission is handled by handleSubmit, not by the button alone (clicking on the button triggers the handlesubmit of the form